### ISSUE TITLE
A synthesis of multiple datasets describing water quality of coastal regions in the Gulf of Mexico 

### ISSUE THREAD

**Issue by zenrabbit** [NCEAS/oss-2017#28]
Author: **Marcus W Beck**
Topic: water quality synthesis

## Context
The Gulf Coast includes several estuaries that are transitional zones between upstream watershed inputs and downstream oceanic areas.  These estuaries range from small, shallow systems in relatively pristine condition, to large, prominent systems such as Tampa and Mobile Bay that have a long management history.  Gulf Coast estuaries have high economic value and are focal points of human activity, yet many of these systems are vulnerable to environmental degradation.  Upstream nutrient inputs can alter rates of primary production such that many systems exhibit symptomatic effects of cultural eutrophication.  Hypoxia events in bottom waters and increased occurrence of harmful algal blooms have been documented in Gulf Coast estuaries and in the larger Gulf of Mexico.  Effects of sea level rise also have the potential to alter water quality conditions and can have direct consequences for shoreline areas.

## Data needs
Data sources describing water quality of coastal areas of the Gulf range from well-established, continuous monitoring programs to discrete data collected at individual sites for specific purposes.  Large systems such as Tampa Bay and Mobile Bay have maintained data collection at fixed sites over several decades.  The National Estuarine Research Reserve System (NERRS) has also collected water quality data since 1995 at five smaller Gulf Coast estuaries.  Other data sources are available but have not been as rigorously maintained as existing monitoring programs, e.g., volunteer water quality samples.  Alternatively, output from simulation models are available at a range of spatial scales, including hypoxia modelling for the Louisiana Coastal Shelf and individual Florida estuaries.  The synthesis of multiple datasets describing water quality of coastal regions in the Gulf of Mexico could have great benefit for our ecological understanding of the region and has the potential to inform future management actions for these dynamic systems. 

## Goals
The proposed project will identify, gather, compile, and analyze existing water quality datasets from Gulf coastal regions to provide a common source of information that can be used by researchers and managers.  The primary focus will be on estuarine systems, but could also include data sources describing input conditions (e.g., USGS stream gage data) or potential downstream endpoints (e.g. Gulf of Mexico hypoxic area). Data sources will likely include publically-available information from online web retrieval services (e.g., http://cdmo.baruch.sc.edu/, http://www.tampabay.wateratlas.usf.edu/) or forward-facing data that are not rigorously organized but can be obtained from data mining (e.g., web scraping, database queries, etc.).  Additional sources of information could include output from simulation studies of specific systems or complete datasets from individual research projects, as they are available.  Once identified and gathered, the data will be formatted and compiled into a common database using a pre-established metadata structure.  

## Significance
The potential impact of this project will depend primarily on the information obtained from analysis.  As a simple example, a common database describing environmental conditions over space and time could be used to compare and contrast differences between systems.  Are there estuaries that have similar temporal dynamics in dissolved oxygen or nutrients that could be managed similarly as compared to others that might require site-specific management plans?  An analysis of temporal variation of diel and seasonal cycles of water quality parameters relative to system characteristics could help identify such information.  A common database could also be used to facilitate the design of systematic monitoring programs for environmental water quality.  Although these programs exist for select locations in the Gulf, several locations are not routinely monitored and those that exist are not readily comparable.  The development or refinement of a monitoring network depends on identifying the quality of information relative to the effort.  A comprehensive database could help quantify this information to determine when and where monitoring could be most effective.

**Comment by esherwoo77** [comment]
This sort of activity has been talked about for many years under GOMA directed efforts, but has never really come to fruition. So, I think it is a very worthwhile effort. The Florida STORET system (http://prodenv.dep.state.fl.us/DearSpa/public/welcome) may also be a source to mine data resources in limitedly sampled regions of FL -- not sure where to start for other states though. I think your definition of "estuary" will be important, as monitoring in my region (Tampa Bay) has traditionally focused in the major bays and FW tributaries. However, little attention has been given to the many minor tidal creek/rivers that discharge to major estuaries or the Gulf proper. We've made a concerted effort to target tidal creeks in the SW FL region to better understand their nutrient dynamics, e.g. 1) http://www.tbeptech.org/TBEP_TECH_PUBS/2016/TBEP_02_16_SW_FL_Tidal_Creeks_Final_Draft_Report_160121.pdf ; 2) http://www.tbeptech.org/TBEP_TECH_PUBS/2008/TBEP_02_08_Tidal_Tributary_Habitat_Initiative_Integrated_Summary_Report.pdf). So, the datasets from these regional research efforts may also be of use for this synthesis project.  See for example: http://www.tbeptech.org/TBEP_TECH_PUBS/2016/TBEP_02b_16_SW_FL_Tidal_Creek_Databases.zip .

**Comment by Yassin22** [comment]
Proposals #28, #25, and #10 complement each other and should be combined. Here are some additional resources that can be useful: http://oceanservice.noaa.gov/hazards/hab/, https://www.epa.gov/ms-htf. Your proposal deals with hypoxia and harmful algae blooms, my proposal deals with harmful algae blooms. Should we explore the impact of other parameters such as metals? Is such data available? I think we can also use existing data to evaluate current hypoxia models?
Our proposals have the potential for significant contribution to scientific knowledge, technically feasible, and use appropriate methods. They would very likely have positive social, economic, and environmental impact.

**Comment by fawda123** [comment]
@esherwoo77 Agreed on all account about the smaller tributaries but I've no idea where to start for data.  We have some data here at EPA but it's project-specific and not comprehensive for the whole gulf.  I suspect our definition of 'estuary' will be shaped in part by where the data are available.

**Comment by esherwoo77** [comment]
Hi @JessicaHenkel , @fawda123 & @pvarelag -- FYI ... I was just forwarded the attached synthesis of GoM Data portals that could be useful to a group project that relates to #22, #28, #32 , #34 .
[GoM_DataPortals.xlsx](https://github.com/NCEAS/oss-2017/files/1001402/GoM_DataPortals.xlsx)
